src/org/labkey/test/WebDriverWrapper.java

```diff
@@ -3994,6 +3994,11 @@ public void setCodeEditorValue(String id, String value)
         _extHelper.setCodeMirrorValue(id, value);
     }
 
+    protected String getCodeEditorValue(String id)
+    {
+        return _extHelper.getCodeMirrorValue(id);
+    }
+
```

Comment:
Use `CodeMirrorHelper` directly. I don't know why the ext helper had this functionality.
```suggestion
    protected String getCodeEditorValue(String id)
    {
        return new CodeMirrorHelper(this, id).getCodeMirrorValue();
    }

```
(Please update `WebDriverWrapper.setCodeEditorValue` in the same way)